### DISCUSSION TITLE
error saying files aren't in gzip format...but they are? -- trimming

### DISCUSSION THREAD

**Discussion by meganewing** [sr320/course-fish546-2021#82] [Q&A]
When I tried to trim my sequences last week I encountered the following error: 

`Failed to process /Volumes/Samsung USB/salmon/Files_Giles_Uploaded_For_Chris/17104-02RT-01-13_S21_L003_R2_001_trim.fastq.gz
java.util.zip.ZipException: Not in GZIP format`

Not sure why I'm getting this error. 
Code where the error occurred (line 4): https://github.com/fish546-2021/Megan-project/blob/main/code/0210-trimSequences.ipynb

**Comment by kubu4** [comment]
Tricky! It wasn't discussed much in the reading, but you can name a file anything you want (e.g. pizza.gz), but the filename isn't always a reflection the file _type_. So, as is almost always the case, I suspect the computer is correct and the file actually is _not_ a GZIP file; despite it name suggesting that it is.

To check, run the following:

`file /Volumes/Samsung USB/salmon/Files_Giles_Uploaded_For_Chris/17104-02RT-01-13_S21_L003_R2_001_trim.fastq.gz`

This will output a description of what the file type is. 

With that being said, it's very possible that this could be related to your [Rsync issue](https://github.com/sr320/course-fish546-2021/discussions/80) - specifically if the file didn't fully transfer or transferred incorrectly. 

Knowing that, I would fix the Rsync issue first, then re-run your `seqtk` command on the freshly transferred files.
